javascript/javascript3/week2/Currency_Converter/index.html

```diff
@@ -0,0 +1,409 @@
+                <option value="SLL" label="Sierra Leonean leone">SLL</option>
+                <option value="SGD" label="Singapore dollar">SGD</option>
+                <option value="SOS" label="Somali shilling">SOS</option>
+                <option value="SLS" label="Somaliland shilling">SLS</option>
+                <option value="ZAR" label="South African rand">ZAR</option>
+                <option value="KRW" label="South Korean won">KRW</option>
+                <option value="SSP" label="South Sudanese pound">SSP</option>
+                <option value="SRD" label="Surinamese dollar">SRD</option>
+                <option value="SEK" label="Swedish krona">SEK</option>
+                <option value="CHF" label="Swiss franc">CHF</option>
+                <option value="LKR" label="Sri Lankan rupee">LKR</option>
+                <option value="SZL" label="Swazi lilangeni">SZL</option>
+                <option value="SYP" label="Syrian pound">SYP</option>
+                <option value="TJS" label="Tajikistani somoni">TJS</option>
+                <option value="TZS" label="Tanzanian shilling">TZS</option>
+                <option value="THB" label="Thai baht">THB</option>
+                <option value="TOP" label="Tongan paʻanga">TOP</option>
+                <option value="PRB" label="Transnistrian ruble">PRB</option>
+                <option value="TTD" label="Trinidad and Tobago dollar">
+                  TTD
+                </option>
+                <option value="TND" label="Tunisian dinar">TND</option>
+                <option value="TRY" label="Turkish lira">TRY</option>
+                <option value="TMT" label="Turkmenistan manat">TMT</option>
+                <option value="TVD" label="Tuvaluan dollar">TVD</option>
+                <option value="UGX" label="Ugandan shilling">UGX</option>
+                <option value="UAH" label="Ukrainian hryvnia">UAH</option>
+                <option value="AED" label="United Arab Emirates dirham">
+                  AED
+                </option>
+                <option value="USD" label="United States dollar">USD</option>
+                <option value="UYU" label="Uruguayan peso">UYU</option>
+                <option value="UZS" label="Uzbekistani soʻm">UZS</option>
+                <option value="VUV" label="Vanuatu vatu">VUV</option>
+                <option value="VES" label="Venezuelan bolívar soberano">
+                  VES
+                </option>
+                <option value="VND" label="Vietnamese đồng">VND</option>
+                <option value="XOF" label="West African CFA franc">XOF</option>
+                <option value="ZMW" label="Zambian kwacha">ZMW</option>
+                <option value="ZWB" label="Zimbabwean bonds">ZWB</option>
+              </select></label
+            >
+          </div>
+        </div>
+        <div class="down-div">
+          <input type="number" class="input-down" aria-label="to convert" />
```

Comment:
Nice with using aria-label. Maybe it would be better to write `aria-label="Converted amount"` Good reading https://www.aditus.io/aria/aria-label/#:~:text=aria%2Dlabel%20is%20an%20attribute,content%20as%20the%20accessibility%20label.